ui/admin/app/components/form/credential/ssh_private_key/index.hbs

```diff
@@ -112,30 +122,36 @@
       {{#if @model.errors.private_key}}
         <F.Error as |E|>
           {{#each @model.errors.private_key as |error|}}
-            <E.Message>{{error.message}}</E.Message>
+            <E.Message data-test-error-message-private-key>
+              {{error.message}}
+            </E.Message>
           {{/each}}
         </F.Error>
       {{/if}}
     </Hds::Form::Textarea::Field>
-    <form.input
-      @type='password'
-      @name='private_key_passphrase'
+
+    <Hds::Form::TextInput::Field
+      name='private_key_passphrase'
       @value={{@model.private_key_passphrase}}
-      @label={{t 'resources.credential.form.private_key_passphrase.label'}}
-      @helperText={{t 'resources.credential.form.private_key_passphrase.help'}}
-      @error={{@model.errors.private_key_passphrase}}
-      @hasVisibilityToggle={{false}}
+      @isInvalid={{@model.errors.private_key_passphrase}}
+      @isOptional={{true}}
+      @type='password'
```

Comment:
None blocking comment: for DX I will suggest the attribute `@type` goes right after `name`. So the developer can see right away the important attributes.

Reply:
@calcaide agreed incoming in next commit